backend/start_local.sh

```
@@ -0,0 +1,23 @@
+
+KEY_FILE=.webui_secret_key
+
+PORT="${PORT:-8080}"
+HOST="${HOST:-0.0.0.0}"
+if test "$WEBUI_SECRET_KEY $WEBUI_JWT_SECRET_KEY" = " "; then
+  echo "Loading WEBUI_SECRET_KEY from file, not provided as an environment variable."
+
+  if ! [ -e "$KEY_FILE" ]; then
+    echo "Generating WEBUI_SECRET_KEY"
+    # Generate a random value to use as a WEBUI_SECRET_KEY in case the user didn't provide one.
+    echo $(head -c 12 /dev/random | base64) > "$KEY_FILE"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)